### ISSUE TITLE
关于训练集与测试集的问题

### ISSUE THREAD

**Issue by xiaomudsg** [wdhudiekou/IMF#12]
作者您好，您在文章中写到During the training phase, we use all images of RoadScene,M3FD, and MSIFT datasets and randomly select 8 image patches of size 256 × 256 to form a batch. During the testing phase, we select 121 images from the RoadScene dataset, 24 images from the TNO dataset, and all images from M3FD and MSIFT datasets as testing data, respectively. 这个意思是测试集用的图片与训练集用的部分图片一样吗？

**Comment by wdhudiekou** [comment]
Thank you for your interest in our work! Due to the small size of the TNO, RoadScene, M3FD, and MSIFT datasets, and the unsupervised nature of our IMF as a registration-fusion method, we conducted testing using all these datasets. It is important to note that during training, deformed infrared images are randomly generated online, ensuring that each patch image's deformation is unique to prevent overfitting. Additionally, the misaligned test set was pre-generated, with different deformations for each image.
